src/viam/sdk/common/linear_algebra.cpp

```diff
@@ -0,0 +1,44 @@
+#include <viam/sdk/common/linear_algebra.hpp>
+
+#include <viam/api/common/v1/common.pb.h>
+
+namespace viam {
+namespace sdk {
+
+double Vector3::x() const {
```

Comment:
Per discussion, these trivial accessors probably do belong as inline. However, let's just add a TODO for that for now because I don't want to muddy the waters about what should/shouldn't be inline at this point. We can always un-inline them later.